portfolio/src/main/webapp/index.html

```diff
@@ -23,18 +23,31 @@ <h1>Korhan Citlak's Portfolio</h1>
       <br>
       <p>My linkedin is <a href="https://www.linkedin.com/in/korhan-citlak-44b345191/">Linkedin</a></p>
       <br> 
+      <p> Your login information is here </p>
+      <ul id="logininfo"></ul>
       <p> Comments will go down here.</p>
-    
-     <br>
+      <br>
       <ul id="history"></ul>
-     <body onload="getComment()">
+
+      <p> Hit the login button below to sign in and logout button to sign out! When your signed in you will be able to see and leave comments! </p>
+      <body onload="loginIn(); getComment();">
+       <form action="/login" method="GET">
+        <div id="hidelogin">
+        <input type="submit" class ="login" value="Login" />
+        </div>
+        <div id="showlogout">
+        <input type="submit" class="logout" value="Logout"/>
+        </div>
+       </form>
+      </body>
+       <div id="hideorshow">
       <form action="/data" method="POST">
        <p>Enter a comment:</p>
        <textarea name="text-input">Enter comment here</textarea>
        <br/><br/>
```

Comment:
I left a comment on the comment servlet, but since someone could use their browser tools to show this form and submit a comment anyway, I think you want to add some security on the backend to prevent logged out users from submitting a comment. If you want to read more about this idea you should check out [`client side vs server side validation`](https://stackoverflow.com/questions/162159/javascript-client-side-vs-server-side-validation).